src/components/tooltips/line-popover.js

```diff
@@ -36,22 +36,28 @@ const useStyles = makeStyles((theme) => ({
     },
 }));
 
-const LinePopover = ({ studyUuid, anchorEl, lineId, loadFlowStatus }) => {
+const LinePopover = ({
+    studyUuid,
+    lineInfos,
+    anchorEl,
+    lineId,
+    loadFlowStatus,
+}) => {
     const currentNode = useSelector((state) => state.currentTreeNode);
     const [lineInfo, setLineInfo] = useState(null);
     const intl = useIntl();
     const classes = useStyles();
     const [localAnchorEl, setLocalAnchorEl] = useState(null);
 
     useEffect(() => {
-        if (lineId !== '') {
+        if (lineInfos) {
+            setLineInfo(lineInfos);
+        } else if (lineId && lineId !== '') {
             fetchLine(studyUuid, currentNode.id, lineId).then((value) => {
                 setLineInfo(value);
             });
-        } else {
-            setLineInfo(null);
```

Comment:
why did you remove that part ? I think it's good practice to empty the state when the data is no longer relevant so that we don't accidentally print old data for a few frames (even if there are others controls that prevents it)